### PR TITLE
Insecure icons

### PR DESCRIPTION
**Pull request by itsluke**
`let's try this again`

### WHY are these changes introduced?

Fixes #1515

### WHAT is this pull request doing?

We have lovely icon we use to talk about security (see pic attached) however due to the fact we're normally telling people their X is secure it's only available with a :heavy_tick: inside. This fixes this by adding an icon to be used to demonstrate risk.

Icons

### CODE REVIEW

File: polaris-icons/icons/InsecureMajor.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+
+<svg viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" d="M9.128.233c-2.37 1.383-5.37 2.33-7.635 2.646-.821.115-1.495.79-1.493 1.62l.001.497c-.03 6.043.477 11.332 9.462 14.903a1.45 1.45 0 001.062 0c8.993-3.571 9.503-8.86 9.473-14.903v-.501c-.001-.828-.674-1.51-1.492-1.638-2.148-.337-5.281-1.274-7.65-2.628a1.733 1.733 0 00-1.728.004zM9 6h2v4H9V6zm2 6H9v2h2v-2z" fill="#D72C0D"/>
+</svg>
```
Comment (alex-page) [inline polaris-icons/icons/InsecureMajor.svg:3]:
```suggestion
<svg viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" d="M9.128.233c-2.37 1.383-5.37 2.33-7.635 2.646-.821.115-1.495.79-1.493 1.62l.001.497c-.03 6.043.477 11.332 9.462 14.903a1.45 1.45 0 001.062 0c8.993-3.571 9.503-8.86 9.473-14.903v-.501c-.001-.828-.674-1.51-1.492-1.638-2.148-.337-5.281-1.274-7.65-2.628a1.733 1.733 0 00-1.728.004zM9 6h2v4H9V6zm2 6H9v2h2v-2z" fill="#5C5F62"/>
</svg>
```

Comment (alex-page) [inline polaris-icons/icons/InsecureMajor.svg]:
```suggestion
<svg viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" d="M9.128.233c-2.37 1.383-5.37 2.33-7.635 2.646-.821.115-1.495.79-1.493 1.62l.001.497c-.03 6.043.477 11.332 9.462 14.903a1.45 1.45 0 001.062 0c8.993-3.571 9.503-8.86 9.473-14.903v-.501c-.001-.828-.674-1.51-1.492-1.638-2.148-.337-5.281-1.274-7.65-2.628a1.733 1.733 0 00-1.728.004zM9 6h2v4H9V6zm2 6H9v2h2v-2z" fill="#5C5F62"/></svg>
```

File: polaris-icons/icons/InsecureMinor.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M19.09 2.678A1 1 0 0018.144 2c-.039 0-3.986-.022-7.692-1.893a1.007 1.007 0 00-.902 0C5.862 1.969 1.895 2 1.856 2a1 1 0 00-.947.678c-1.367 4.03-1.19 7.825.515 10.97 1.647 3.042 4.76 5.404 8.328 6.32a1 1 0 00.497 0c3.567-.916 6.68-3.278 8.328-6.32 1.703-3.145 1.88-6.94.513-10.97zm-2.273 10.02c-1.349 2.49-3.884 4.444-6.816 5.268-2.934-.824-5.469-2.778-6.818-5.268-1.343-2.48-1.54-5.485-.585-8.727 1.36-.088 4.383-.444 7.403-1.858 3.018 1.414 6.04 1.77 7.401 1.858.958 3.242.758 6.248-.585 8.727zM9 6h2v4H9V6zm2 6H9v2h2v-2z" fill="#D72C0D"/></svg>
```
Comment (alex-page) [inline polaris-icons/icons/InsecureMinor.svg]:
```suggestion
<svg viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M19.09 2.678A1 1 0 0018.144 2c-.039 0-3.986-.022-7.692-1.893a1.007 1.007 0 00-.902 0C5.862 1.969 1.895 2 1.856 2a1 1 0 00-.947.678c-1.367 4.03-1.19 7.825.515 10.97 1.647 3.042 4.76 5.404 8.328 6.32a1 1 0 00.497 0c3.567-.916 6.68-3.278 8.328-6.32 1.703-3.145 1.88-6.94.513-10.97zm-2.273 10.02c-1.349 2.49-3.884 4.444-6.816 5.268-2.934-.824-5.469-2.778-6.818-5.268-1.343-2.48-1.54-5.485-.585-8.727 1.36-.088 4.383-.444 7.403-1.858 3.018 1.414 6.04 1.77 7.401 1.858.958 3.242.758 6.248-.585 8.727zM9 6h2v4H9V6zm2 6H9v2h2v-2z" fill="#5C5F62"/></svg>
```

**Review by alex-page — CHANGES_REQUESTED** [review]
The colors and formatting should be cleaned up to use the default icon approach. That should fix the tests. I left some suggestions but this is looking great @itsluke.

@sarahill @nayeob-kim can you take a look at the icon design. It looks good to me but just want to confirm it's up to expectations.

**Comment by itsluke** [comment]
Thanks for the help with the colours and formatting @alex-page and support with the PR.

**Review by alex-page — APPROVED** [review]
Would love a designer to review this before shipping. Going to ping folks on our team.

**Comment by alex-page** [comment]
@itsluke it seems like CI is failing as the SVG could have a regression in some environments. You cannot use the `clip-rule` attribute on a path so you will need to recreate the SVG without it.

```diff
SVG Contents: packages/icons/InsecureMinor.svg › only has <path>s that only use the [d, fill, fill-rule] attributes

expect(received).toStrictEqual(expected) // deep equality

- Expected  - 1
+ Received  + 3

- Array []
+ Array [
+   "<path fill-rule=\"evenodd\" clip-rule=\"evenodd\" d=\"M19.09 2.678A1 1 0 0018.144 2c-.039 0-3.986-.022-7.692-1.893a1.007 1.007 0 00-.902 0C5.862 1.969 1.895 2 1.856 2a1 1 0 00-.947.678c-1.367 4.03-1.19 7.825.515 10.97 1.647 3.042 4.76 5.404 8.328 6.32a1 1 0 00.497 0c3.567-.916 6.68-3.278 8.328-6.32 1.703-3.145 1.88-6.94.513-10.97zm-2.273 10.02c-1.349 2.49-3.884 4.444-6.816 5.268-2.934-.824-5.469-2.778-6.818-5.268-1.343-2.48-1.54-5.485-.585-8.727 1.36-.088 4.383-.444 7.403-1.858 3.018 1.414 6.04 1.77 7.401 1.858.958 3.242.758 6.248-.585 8.727zM9 6h2v4H9V6zm2 6H9v2h2v-2z\" fill=\"#5C5F62\"/>",
+ ]

   96 |         expect(
   97 |           nodeSources(nodesWithDisallowedAttributes, iconSource),
>  98 |         ).toStrictEqual([]);
      |           ^
   99 |       });
  100 |
  101 |       it('only has <polygon>s that only use the [fill, points] attributes', () => {
```

**Comment by heyjoethomas** [comment]
@itsluke Is there a reason the major version is filled and the minor is stroked?

**Review by chloerice — CHANGES_REQUESTED** [review]
Hey @itsluke, it's been a while!! 👋🏽  This is looking great, but I think we should align the exclamation point with [the other warning icons in the system](https://polaris-icons.shopify.com/?q=warn).

**Comment by zecarlostorre** [comment]
image

We already have this icon in our set, it's called: [**FraudProtectUnprotectedMajor**](https://polaris-icons.shopify.com/?icon=FraudProtectUnprotectedMajor&q=fra)

If there's a reason to introduce an outlined version, I'd recommend making the exclamation mark rounded, to match our set.

image

**Comment by alex-page** [comment]
@itsluke this PR could be an update to the existing icon to add keywords as I too struggled to find the existing icon. These are great keywords.

```
  - unprotected
  - insecure
  - risk
  - danger
  - unverified
```

**Comment by heyjoethomas** [comment]
> image
> 
> We already have this icon in our set, it's called: [**FraudProtectUnprotectedMajor**](https://polaris-icons.shopify.com/?icon=FraudProtectUnprotectedMajor&q=fra)
> 
> If there's a reason to introduce an outlined version, I'd recommend making the exclamation mark rounded, to match our set.
> 
> image

Good catch Jose.

**Comment by itsluke** [comment]
> We already have this icon in our set, it's called: [**FraudProtectUnprotectedMajor**](https://polaris-icons.shopify.com/?icon=FraudProtectUnprotectedMajor&q=fra)

@zecarlostorre Thanks for that catch that's great news!  @chloerice & @alex-page appreciate the comments. 

I'll clean this up and slim down the commit to the metadata improvements

**Pull request closed without merging**